main/integration-tests/support/setup.js

```diff
@@ -82,13 +83,13 @@ class Setup {
 
   async getDefaultIndexId() {
     const adminSession = await this.defaultAdminSession();
-    const defaultProject = await adminSession.resources.projects.project(this.gen.defaultProjectId()).get();
+    const defaultProject = await adminSession.resources.projects.project(this.defaults.projectId).get();
```

Comment:
Doing this.defaults.projectId (inside this function) does require that getDefaultIndexId() be called at a specific sequence when building the defaults. I would suggest that you make the function as follows:
`async getDefaultIndexId(projectId) { ... }`
This will eliminate the need to use this.defaults when we are still initializing it.

Reply:
Combined in one method as discussed